packages/fxa-shared/db/models/auth/linked-account.ts

```diff
@@ -47,13 +47,17 @@ export class LinkedAccount extends BaseAuthModel {
     });
   }
 
-  static async deleteLinkedGoogleAccount(uid: string, id: string) {
+  static async deleteLinkedGoogleAccount(uid: string) {
     await LinkedAccount.query()
```

Comment:
Should the number records deleted be returned? This maybe this outside the scope of this PR since this functionality already existed, but at the API level, it seems important to know if the deleting the linked account had an effect.